CHANGELOG.md

```diff
@@ -0,0 +1,41 @@
+
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+## Unreleased
+
+## Bootes - 16-08-2019
+### Added
+- New plugin architecture for the following Asperathos components:
+   - Monitor
+   - Manager
+   - Visualizer
+   - Controller
+- Persistent plugins for the following Asperathos components:
+   - Monitor
+   - Manager
+   - Visualizer
+   - Controller
+- PID scalling strategy at Asperathos Controller 
+
+- New tests for kubejobs plugin at Asperathos Monitor
+
+### Changed
+- Refactoring of the kubejobs plugin at Asperathos Manager
+
+###  Removed
+- Unused legacy plugins that were not working anymore
```

Comment:
Never good to say something is "not working anymore" :) We should often say things are no longer supported, or not supported at the moment. That way, I would rephrase it to "... plugins that are not supported at the moment."